src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -2905,7 +2905,7 @@ static void window_ride_construction_update_widgets(rct_window *w)
     window_ride_construction_widgets[WIDX_U_TRACK].type = WWT_EMPTY;
     window_ride_construction_widgets[WIDX_O_TRACK].type = WWT_EMPTY;
 
-    bool brakesSelected = _selectedTrackType == TRACK_ELEM_BRAKES || _currentTrackCurve == (0x100 | TRACK_ELEM_BRAKES);
+    bool brakesSelected = _selectedTrackType == TRACK_ELEM_BRAKES || _currentTrackCurve == (0x100 | TRACK_ELEM_BRAKES) || _selectedTrackType == TRACK_ELEM_BLOCK_BRAKES || _currentTrackCurve == (0x100 | TRACK_ELEM_BLOCK_BRAKES);
```

Comment:
Can you name the `0x100` flag?

Reply:
is there an enum somewhere for those flags. I am honestly not even sure what 0x100 is the flag for.

Reply:
This flag is used throughout the RideConstruction.cpp code. It triggers this code
https://github.com/OpenRCT2/OpenRCT2/blob/fe76c807aa98696a30197f239fe6fb0e31adfab3/src/openrct2-ui/windows/RideConstruction.cpp#L730-L743
https://github.com/OpenRCT2/OpenRCT2/blob/fe76c807aa98696a30197f239fe6fb0e31adfab3/src/openrct2-ui/windows/RideConstruction.cpp#L2092-L2099

I think a rename should be a different pull request.

Reply:
If you're not entirely sure what the flag does, you can leave it as is.